src/components/Reports/TableCellRenderer.tsx

```diff
@@ -13,7 +13,17 @@ export default function ReportsCellRenderer(props: RendererProps<TableRowType>):
     const reportLocation = {
       pathname: APP_PATHS.REPORTS_VIEW.replace(':reportId', row.id.toString()),
     };
-    return <Link to={reportLocation.pathname}>{`${row.year}-Q${row.quarter}` as React.ReactNode}</Link>;
+
+    let reportName = `${row.year}-Q${row.quarter}`;
+    if (row.projectName) {
```

Comment:
No feature flag check here since the reports can only be created for a project if someone has the ability to create projects in the first place.